koptional-reactor-extensions/src/test/kotlin/com/gojuno/koptional/reactor/FluxSpec.kt

```diff
@@ -0,0 +1,33 @@
+package com.gojuno.koptional.reactor
+
+import com.gojuno.koptional.None
+import com.gojuno.koptional.Optional
+import com.gojuno.koptional.Some
+import org.jetbrains.spek.api.Spek
+import org.jetbrains.spek.api.dsl.it
+import org.junit.platform.commons.annotation.Testable
+import reactor.core.publisher.Flux
+import reactor.test.StepVerifier
+
+@Testable
```

Comment:
This annotation allows running the specification in IDE via the green arrow (without additional plugins). Ideally the parent `Spek` should be annotated with it but it isn’t.